adagrams/game.py

```diff
@@ -1,11 +1,98 @@
+import random
+
 def draw_letters():
-    pass
+    letter_pool =["A", "A", "A", "A", "A", "A", "A", "A", "A", "B", "B", "C", \
+        "C", "D", "D", "D", "D", "E", "E", "E", "E", "E", "E", "E", "E", "E", \
+        "E", "E", "E", "F", "F", "G", "G", "G", "H", "H", "I", "I", "I", "I", \
+        "I", "I", "I", "I", "I", "J", "K", "L", "L", "L", "L", "M", "M", "N", \
+        "N", "N", "N", "N", "N", "O", "O", "O", "O", "O", "O", "O", "O", "P", \
+        "P", "Q", "R", "R", "R", "R", "R", "R", "S", "S", "S", "S", "T", "T", \
+        "T", "T", "T", "T", "U", "U", "U", "U", "V", "V", "W", "W", "X", "Y", \
+        "Y", "Z"]
+    
+    letter_bank = []
+
+    letter_bank = random.sample(letter_pool, 10)
+    
+    return letter_bank
+
+def make_word_upper_list(word):
+    user_input_list = []
+    word = word.upper()
+    for letter in word:
+        user_input_list.append(letter)
+    return user_input_list
 
 def uses_available_letters(word, letter_bank):
-    pass
+    is_valid = False
+
+    user_input = make_word_upper_list(word)
```

Comment:
You can simplify this to:

```suggestion
    user_input = list(word.upper())
```

Though it doesn't need to be a list to be able to iterate over it.